src/string_pool.cpp

```diff
@@ -29,6 +29,7 @@
 /*static*/ std::string_view pool_string::default_view("");
 
 #define ORC_PRIVATE_FEATURE_PROFILE_POOL_MEMORY() (ORC_PRIVATE_FEATURE_TRACY() && 0)
+#define ORC_PRIVATE_FEATURE_PROFILE_POOL_MUTEXES() (ORC_PRIVATE_FEATURE_TRACY() && 1)
```

Comment:
I assume you are keeping this enabled on purpose?

Reply:
Yeah for now. Tracy should be turned off by default so shouldn't impact releases anyhow.